src/Quizzer.API.BL.UnitTests/AnswerFacadeTest.cs

```diff
@@ -0,0 +1,112 @@
+
+            // Act
+            IEnumerable<AnswerDetailModel> models = answerFacade.GetAllAnswers();
+
+            // Assert
+            _uowFactoryMock.Verify(factory => factory.Create(), Times.Once);
+            _uowMock.Verify(uow => uow.GetRepository<AnswerEntity>(), Times.Once);
+            _answerRepositoryMock.Verify(rep => rep.GetAll());
+        }
+
+        [Theory]
+        [InlineData(-1)]
+        [InlineData(-80)]
+        [InlineData(-11981)]
+        [InlineData(-110)]
+        [InlineData(-2147483648)]
+        [InlineData(-2)]
+        public void Save_Answer_With_Bad_Order_ThrowsException(int badOrder) {
+            // Arrange 
+            var answerDetailModel = new AnswerDetailModel { Id = Guid.Empty, Text = "", Order = badOrder };
+            AnswerFacade answerFacade = GetFacadeWithForbiddenDependencyCalls();
+
+            // Act & Assert
+            Assert.Throws<ArgumentException>(() => answerFacade.Save(answerDetailModel));
+        }
+
+        [Theory]
+        [InlineData(null, null)]
+        [InlineData(null, "")]
+        [InlineData(null, "              ")]
+        [InlineData(null, "\t")]
+        [InlineData("", null)]
+        [InlineData("", "")]
+        [InlineData("", "       ")]
+        [InlineData("", "\t")]
+        [InlineData("         ", null)]
+        [InlineData("         ", "")]
+        [InlineData("         ", "     ")]
+        [InlineData("         ", "\t")]
+        [InlineData("\t", null)]
+        [InlineData("\t", "")]
+        [InlineData("\t", "    ")]
+        [InlineData("\t", "\t")]
+        public void Save_Answer_With_Bad_Text_Or_Url_ThrowsException(string text, string url) {
+            // Arrange 
+            var answerDetailModel = new AnswerDetailModel { Id = Guid.Empty, Text = text, Order = 5, PictureUrl = url };
+            AnswerFacade answerFacade = GetFacadeWithForbiddenDependencyCalls();
+
+            // Act & Assert
+            Assert.Throws<ArgumentException>(() => answerFacade.Save(answerDetailModel));
+        }
+
+        [Theory]
+        [InlineData(null, "url2")]
+        [InlineData("", "url1")]
+        [InlineData("         ", "3url")]
+        [InlineData("\t", "url6")]
+        [InlineData("Some name", null)]
+        [InlineData("I am not empty", "")]
+        [InlineData("got this text?", "            ")]
+        [InlineData("feel free?", "\t")]
+        public void Save_Answer_With_Bad_Text_Or_Url_Expected_Call_Save(string text, string url) {
+            // Arrange 
+            var answerDetailModel = new AnswerDetailModel { Id = Guid.Empty, Text = text, Order = 5, PictureUrl = url };
+            AnswerFacade answerFacade = GetFacadeWithForbiddenDependencyCalls();
+
+            _mapperMock.Setup(mapper => mapper.Map<AnswerEntity>(It.IsAny<AnswerDetailModel>())).Returns(
+                new AnswerEntity { Id = Guid.NewGuid(), Text = "empty", IsCorrect = false });
```

Comment:
I would break the line on `.Returns`, it looks better

Reply:
done